include/todo_lib/TodoModel.h

```diff
@@ -0,0 +1,34 @@
+#ifndef TODOMODEL_H
+#define TODOMODEL_H
+
+#include <string>
+
+class TodoModel {
+public:
+    // attributes
+    unsigned int id;
+    bool status = 1;
+    std::string title;
+    std::string description = "Add description... with <code for description>";
+
+    //constructors
+    TodoModel();
+    TodoModel(int id, std::string title);
+    TodoModel(int id, std::string title, std::string description);
```

Comment:
id initialization has to be done by the constructor, not to be passed as parameter